org.mixedrealitytoolkit.uxcore/Experimental/List/VirtualizedScrollRectList.cs

```diff
@@ -394,7 +387,7 @@ private void Initialize()
 
         private void InitializePool()
         {
-            // Support resetting everything from OnValidate
+            // Support resetting everything from Reset
```

Comment:
See other comment....please update this comment once the Reset funciton is renamed